conformance_tests/tools/debug/src/test_debug.hpp

```diff
@@ -88,7 +88,7 @@ class ProcessLauncher {
 
   bp::child launch_process(debug_test_type_t test_type,
                            ze_device_handle_t device, bool use_sub_devices,
-                           std::string module_name) {
+                           const char *module_name) {
```

Comment:
why is this needed? All of the other overloads of launch_process take input as std::string